here/dir.go

```diff
@@ -39,20 +40,30 @@ func Dir(p string) (Info, error) {
 				return i, err
 			}
 
+			if strings.Contains(es, "can't load package: package .") {
+				if _, err := os.Stat(fmt.Sprintf("%s/go.mod", p)); err == nil {
```

Comment:
This underlying issue isn't a missing `go.mod` file, but rather, not `.go` files at all and no `go.mod`. The test should check for the existence of `.go` files as well. Go requires at least one `.go` file *or* a `go.mod`.

Reply:
Ok I will try to cover all the case around the Dir function itself, with tests.